client-spark/spark3/src/test/java/org/apache/spark/shuffle/DelegationRssShuffleManagerTest.java

```diff
@@ -78,6 +78,7 @@ public void testCreateFallback() throws Exception {
     conf.set(RssSparkConfig.RSS_DYNAMIC_CLIENT_CONF_ENABLED.key(), "false");
     conf.set(RssSparkConfig.RSS_ACCESS_ID.key(), "mockId");
     conf.set(RssSparkConfig.RSS_ENABLED.key(), "true");
+    conf.set(RssSparkConfig.RSS_STORAGE_TYPE.key(), "MEMORY_LOCALFILE");
```

Comment:
Why do we need to add this?

Reply:
This test further down asserts an exception is thrown. Moving `RssSparkShuffleUtils.validateRssClientConf(sparkConf)` up in the `RssShuffleManager` constructor throws a different exception if the storage type is not set.

Reply:
Coordinator can apply the storage type to the client. Will this pull request affect this feature.

Reply:
This test calls into `new RssShuffleManager(sparkConf, false)`, which mimics creating the shuffle manager on an executor, and executors do not fetch dynamic config, so the given `sparkConf` must have the storage type set for the test.

The logic of applying config is unchanged, this only happens earlier. I presume there are tests for this use case, which should check the desired behaviour is preserved.